modelmatrix-core/src/main/scala/com/collective/modelmatrix/db/DatabaseConfig.scala

```diff
@@ -0,0 +1,56 @@
+package com.collective.modelmatrix.db
+
+import com.typesafe.config.{Config, ConfigFactory}
+import slick.driver.{H2Driver, JdbcProfile, PostgresDriver}
+
+/**
+ * Created by jpocalan on 7/27/15.
+ */
+// TODO: replace by some sort of enum
+// Differentiate between the different types of databse supported
+class DatabaseConfig(n: String, dc: String, sd: JdbcProfile) {
+  val name = n;
+  val driverClass = dc;
+  val slickDriver = sd;
+}
+
+// database configuration wrapper class that read the configuration files and
+// determines the proper slick driver to use
+case class DatabaseConfigWrapper(configFilePath: String = "") {
+  val PG = new DatabaseConfig("pg", "org.postgresql.Driver", PostgresDriver)
+  val H2 = new DatabaseConfig("h2", "org.h2.Driver", H2Driver)
```

Comment:
I think that private[DatabaseConfigWrapper] equivalent to just private, so privte[this] will be better in this case